stock_release_channel/views/stock_release_channel_views.xml

```diff
@@ -148,8 +224,20 @@
                                     <div class="o_kanban_card_header_title">
                                         <div class="o_primary">
                                             <field name="name" />
+                                            <field name="state" invisible="1" />
```

Comment:
I think it should be outside the template in the list of fields to pre-load in the kanban (without invisible)